core/src/main/java/org/javarosa/xform/parse/XFormParser.java

```diff
@@ -324,20 +327,24 @@ private void initState() {
 
     public XFormParser(Reader reader) {
         _reader = reader;
+        extensionParsers = new Vector<>();

```

Comment:
Not sure if J2ME supports empty <>'s. Might want to check or explicitly write out ```<QuestionExtensionParser>```